tests/components/abode/conftest.py

```diff
@@ -0,0 +1,24 @@
+"""Configuration for Abode tests."""
+import abodepy.helpers.constants as CONST
+import pytest
+
+from tests.common import load_fixture
+
+
+@pytest.fixture(autouse=True)
+def requests_mock_fixture(requests_mock):
+    """Fixture to provide a requests mocker."""
+    # Mocks the login response for abodepy.
+    requests_mock.post(CONST.LOGIN_URL, text=load_fixture("abode_login.json"))
+    # Mocks the logout response for abodepy.
+    requests_mock.post(CONST.LOGOUT_URL, text=load_fixture("abode_logout.json"))
```

Comment:
This might be unnecessary since the only time I'm calling `logout`, I'm patching it within the test itself.

Reply:
You decide what you think is best here.